superset/assets/javascripts/explorev2/reducers/exploreReducer.js

```diff
@@ -90,6 +87,7 @@ export const exploreReducer = function (state, action) {
           chartUpdateEndTime: null,
           chartUpdateStartTime: now(),
           triggerQuery: false,
+          queryRequest: action.queryRequest,
```

Comment:
my thinking is that i would prefer to put an object with methods on it into the state rather than move the async calls to the component. both are not ideal best practices, but putting an object with methods into the state seems cleaner to me. @vera-liu what do you think at this point?